### DISCUSSION TITLE
Note about disabling submit/cancel buttons

### DISCUSSION THREAD

**Discussion by ramoneguru** [twilio-labs/paste#558] [General] [answered]
In the Paste docs (https://paste.twilio.design/form-elements/input/#validation) it mentions to not disable submit/cancel buttons. Do we need to be calling that out during design reviews? I'm currently working on a form that has me disabling said buttons.

```
Don't prevent form submission by disabling the submit button. An error message can give more information than a disabled button can to help users figure out which fields are invalid.
```

**Comment by serifluous** [comment, marked as answer]
Yes! We'd love if you called that out during reviews, at least to start a discussion about whether it should be disabled or not for your specific case.

There _may_ be a valid reason the button is disabled (and the designer may have already chatted with us and other designers about it). For example, where there's only a single input or it's overwhelmingly obvious how to enable the button, it's less risky to have the button disabled.

The guideline on the Paste website is basically the least risky thing to do.
